apps/fulltextsearch.sh

```diff
@@ -47,13 +49,13 @@ else
         fi
     done
     # Removal Docker image
-    docker_prune_this "$nc_fts"
+    docker_prune_this "$opens_fts"
```

Comment:
what about also resetting the fts_os-data volume?

Reply:
solved by 61bd632f40c6bdd2a20cec8075104f0aa1e27b2e